src/handler.jl

```diff
@@ -112,6 +112,9 @@ end
 
 function Memento.emit(handler::CloudWatchLogHandler, record::Record)
     dt = isdefined(record, :date) ? record.date : Dates.now(tz"UTC")
+    # Memento@1.3 switched to only storing DateTime in order to make TimeZones.jl an
+    # optional dependency, but AWS requries timestamps in UTC, so we need a ZDT
+    dt = isa(dt, ZonedDateTime) ? dt : astimezone(ZonedDateTime(dt, localzone()), tz"UTC")
```

Comment:
```suggestion
    dt = isa(dt, DateTime) ? astimezone(ZonedDateTime(dt, localzone()), tz"UTC") : dt
```
It's technically possible for someone's record to have e.g. a millisecond value as this property, or another date type (maybe UTCDateTime eventually!). I think it's better to only address the known problematic case.